cmd/head/head.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"log"
+)
+func check(e error) {
+	if e != nil {
+		log.Fatal(e)
+	}
+}
+
+var(
+	numberOfLines int
+)
+
+func main() {
+	headCmd := flag.NewFlagSet("head", flag.ExitOnError)
+	headCmd.IntVar(&numberOfLines,"n", 10, "numberOfLines")
+	headCmd.Parse(os.Args[1:])
+	path := headCmd.Args()[0]
+	absPath, err := filepath.Abs(path)
```

Comment:
you don't need to get the absolute path `os.open` would work fine with the given path